Code/UnitTests/RendererTest/Basics/PipelineStates.cpp

```diff
@@ -778,7 +778,7 @@ void ezRendererTestPipelineStates::Timestamps()
 
   if (m_iFrame > 2 && !m_bTimestampsValid)
   {
-    if (m_bTimestampsValid = m_pDevice->GetTimestampResult(m_timestamps[0], m_GPUTime[0]).Succeeded() && m_pDevice->GetTimestampResult(m_timestamps[1], m_GPUTime[1]).Succeeded())
+    if ((m_bTimestampsValid = m_pDevice->GetTimestampResult(m_timestamps[0], m_GPUTime[0]).Succeeded() && m_pDevice->GetTimestampResult(m_timestamps[1], m_GPUTime[1]).Succeeded()))
```

Comment:
The bracket seems redundant.

Reply:
No, it is not. It silences a warning in clang, letting clang now that we indeed want to do an asignment here (and not a ==)